_posts/2020-05-06-welcome-to-webarkit.markdown

```diff
@@ -6,4 +6,13 @@ description: The welcome article of webarkit.org blog
 author: Walter Perdan
 image: ./resources/webarkit-logo-social.jpg
 ---
-Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+We are pleased to present our new WebARKit Blog page. In this blog we will keep you updated on the news that has happened within the WebArKit community. We will generally publish at least one article every three months and if possible more often to keep you updated.
+What has happened so far in the WebARKit community?
+I personally started creating a simplified version of Artoolkit with only NFT support, it's called WebARKitLib. The code is pretty much the same but it's very light because all unnecessary parts for Natural Feature Tracking have been removed. The final code is only 19.7 mb against the almost 500 mb of the old repository. I have used this library with JsartoolkitNFT and ArNFT with excellent results.
+Thosten Bux instead added partial support for NFT markers and 2d tracking to the ArtoolkitX.js project, it is essential to complete the implementation for the NFT but we are on the right track.
+Worth noting is the great work that Thorsten Bux and Daniel Fernandes have done for the online version of the nft creator, a very useful tool for those who want to work with this type of marker.
+Unfortunately as regards WebARStudio, some improvements have been made but not yet for the NFT markers, it is in fact necessary to develop the code to support the version with Web Worker required for Mobile devices. I hope to accomplish this upgrade in a timely, because it is a long awaited feature …
+An interesting mention should be made for the code developed by Itamayo for the rotation of the camera and a renderer for graphic objects developed by Daniel Fernandes based on CSS3D rendering by Threejs, many interesting possibilities open up with these two projects !!
+Finally, the icing on the cake is the incredible work done by hiukim, he has in fact translated the Artoolkit code concerning the NFT with gpu.js, creating a version completely made for the webGPU!! For sure this will be the future of augmented reality... Obviously the code needs to be improved and most likely the algorithm needs to be changed. But in the meantime: good hiukim! Great!
+If you are interested, follow us and subscribe to our web page or follow our Twitter profile, and if you want to get in touch with us to participate and contribute to the project, let's meet in our slack channel, we are open for collaborations.
```

Comment:
Link to Slack channel

Reply:
i will add it!

Reply:
done with https://github.com/webarkit/webarkit.github.io/pull/4/commits/dc3073aa96d8f4ef63f4398c276acd2274ef6246